websites/B/Board Game Arena/games/D/dudo.ts

```diff
@@ -0,0 +1,40 @@
+import { GamePresence } from "..";
+import {
+	getActivePlayerId,
+	getCurrentGameState,
+	getCurrentGameStateType,
+	getPlayerAvatar,
+	getPlayerData,
+	getPlayerScore,
+	getUserPlayerId,
+} from "../../util";
+
+const dudo: GamePresence = {
+	logo: "https://i.imgur.com/7x5X289.png",
```

Comment:
dudo perudo? is this correct

Reply:
Yes, this is correct. There are a lot of them like this where the id of the game doesn't seem to match the name on the logo.